src/py/pyodide/_package_loader.py

```diff
@@ -360,23 +358,3 @@ def init_loaded_packages() -> None:
     """
     for dist in importlib_distributions():
         setattr(loadedPackages, dist.name, get_dist_source(dist))
-
-
-def sub_resource_hash(sha_256: str) -> str:
```

Comment:
Probably this line should also be removed.

https://github.com/pyodide/pyodide/blob/09c7251d5677bcccb416f2f5186c6bff6a15f59f/conftest.py#L36